sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/EmbeddingItem.java

```diff
@@ -62,8 +91,13 @@ public int getPromptIndex() {
     @Override
     public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
         jsonWriter.writeStartObject();
-        jsonWriter.writeStringField("embedding", this.embeddingBase64);
-        jsonWriter.writeIntField("index", this.promptIndex);
+        jsonWriter.writeStringField("object", "embedding");
```

Comment:
Why is this added?

Reply:
(1) OpenAI's payload has this field "object": "embedding"

image

[API Reference - OpenAI API](https://platform.openai.com/docs/api-reference/embeddings/object)



(2) Generated code has used the same pattern:

image

[[OpenAI-Inference] Inference SDK support for Azure OpenAI `v2024-07-01-preview` service release by jpalvarezl · Pull Request #41210 · Azure/azure-sdk-for-java (github.com)](https://github.com/Azure/azure-sdk-for-java/pull/41210/files#diff-1158b0e826e3bd10ddd27567b1fa563dfda502395a7a52d02a95f56575a7b0b2R25)

Reply:
Is this a new field introduced in the latest version? It was not there in previous versions.

Reply:
new field removed.